src/components/Task.vue

```diff
@@ -56,12 +65,31 @@ export default defineComponent({
     NTag,
     TreeViewAlt
   },
+  methods:{
+    getAssigneeInitials(assignee: string) {
+      if (assignee) {
```

Comment:
This is a bit of a code smell. This is a truthy check which means it looks at null, undefined and also things like blank strings. If this value is nullable/undefined then the parameter should specify that. If it is blank string then best to look for that